src/test/resources/results/result.yaml

```diff
@@ -0,0 +1,4 @@
+!!edu.washington.cs.skeleton.util.ResultConfig
```

Comment:
Do you want to include them in the source code? They seem generated by tests. If you want to do a regression check, you may include the output as part of your repo.